content/applications/inventory_and_mrp/inventory/warehouses_storage/replenishment/reordering_rules.rst

```diff
@@ -59,25 +56,34 @@ If a :abbr:`BoM (Bill of Materials)` does not already exist for the product, sel
 :guilabel:`New` to configure a new :abbr:`BoM (Bill of Materials)`.
 
 .. image:: reordering_rules/bom-smart-button.png
-   :align: center
    :alt: The Bill of Materials smart button on a product form.
 
 Create new reordering rules
 ===========================
 
-To create a new reordering rule, navigate to :menuselection:`Inventory app --> Configuration -->
-Reordering Rules`, then click :guilabel:`New`, and fill out the new line as follows:
+To create a new reordering rule, navigate to
+:menuselection:`Inventory app --> Operations --> Replenishment`, then click :guilabel:`New`,
+and fill out the following fields for the new reordering rule line item:
 
 - :guilabel:`Product`: The product that is replenished by the rule.
-- :guilabel:`Location`: The location where the product is stored.
+- :guilabel:`On Hand`: The number of units currently available in inventory.
+- :guilabel:`Forecast`: The number of units expected to be available in inventory after all orders
+  are taken into account.
```

Comment:
I like that you added more context to the UI here. I think for this particular operation, the **On Hand** and **Forecast** values are not editable for a **New** rule line (on my screen it's grayed out and the cursor goes right to the **Route** field, screenshot below)...

![Screenshot 2024-12-18 at 4 54 46 PM](https://github.com/user-attachments/assets/0963aff9-ec53-47ee-b4fc-b5f3ed0d76dc)

...which, makes sense since **Route** **Min Quantity** **Max Quantity** and **To Order** are the inputs directly tied to the RR, whereas the other two are more closely tied to making an inventory adjustment.

My suggestion is to remove these list items, or optionally add a **see also** admonition that points to any relevant docs on how to make an inventory adjustment.